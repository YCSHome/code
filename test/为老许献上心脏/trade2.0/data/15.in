400 500 500 
129 307232412393460
361 306126225668829
194 87859604039412
224 555323579805326
205 799254858563891
317 142121238765379
243 956667532687538
156 711884168635191
175 358911373305263
315 126904053158315
380 303227843683045
336 751786225744761
298 316038433813475
65 494326528443048
394 282920108616158
314 934279946027776
38 435824780268842
237 377478316979331
255 455709935329220
53 272426799345844
29 662650943214000
36 577518938799987
25 63397373469110
336 246695097195105
285 211879016499130
164 307710165686734
163 768106960467115
277 316624863522370
252 437406417470492
333 504440063155326
246 222128944593376
122 267104761312929
56 872453352199091
165 277788669547694
321 33713896188669
178 197910767351212
53 723500064247189
290 542458209679625
171 219501589161149
58 905384720206472
110 500566991545378
280 272502155178984
302 936749678361259
215 66486438286879
274 527787775364774
270 692385317557200
112 144709309363968
268 831104055053458
143 644163929120000
188 909973248218030
252 453784645166566
16 95029332282550
9 801447038918662
373 666404576017750
254 888958099060074
384 253223492701369
155 667596153649719
380 763037150326029
296 441636256788988
220 264933889571067
169 244467408489496
73 431414955330448
220 492961325251518
42 330658888716632
46 447433799461215
305 87809018293960
368 963700104757033
110 230190139767353
264 862004966702801
268 464668601689186
325 668301060670834
68 545862023428863
77 215678303446025
386 362066513005838
397 489392377314935
363 306507418934536
111 742659202023738
226 660107032273594
367 961283717319317
276 384273006838207
40 435250595002024
229 200485983886210
327 731621434805439
381 467004321023279
266 554053407621329
78 391105509790701
377 179804915175170
338 327247746014675
227 488956868913501
356 414570534721659
138 270961567055824
83 678955952641755
335 452945097930321
266 660224335873818
322 673851923989512
20 31124632577839
173 199048150083159
365 333582020183059
77 613846707253546
195 833392946308849
307 354811016466921
313 333803163018780
148 475738028433674
357 540890454648701
155 37009525712824
268 843047545239248
270 769002452081083
124 798732061644403
292 103303527433861
361 644575282806331
375 693214801148661
235 803903543282863
17 445625623356402
323 244702756680443
40 338589031507927
302 730258441607922
254 114765064541851
262 164220599852073
223 53533220581848
347 517133191392755
96 488966678458077
338 424545106227819
21 451920221161889
389 518680080438259
18 750652524782113
102 262240431731085
147 274631261787362
265 264566679247227
12 646698503467771
137 350737971721903
282 73753086940079
252 365371321375310
65 313956755426267
69 48961918581120
227 629713094461760
335 341526147802761
155 323935034150064
130 390009861609038
53 401278241206778
347 264198889543039
138 561205419614004
129 179603626942814
323 579617717948332
83 386711253107646
21 263623978987371
31 93764694482138
96 437677071161946
66 179559857333784
291 884249813175321
317 337110377973239
252 595985194370537
210 301254933019177
57 109607470153930
197 259471413538249
248 964360665083907
133 87903408650787
334 600699297628536
141 859576058246728
27 675468858779958
70 532341831628843
274 101954365383295
363 960475144133031
205 629943766452830
105 993553540174440
220 162127107858652
379 156288103622133
174 497050367147137
366 111964277821533
141 931191696560759
275 434225328425659
42 378015471589025
347 853627088463978
102 963175593941602
129 804865698980207
27 413673842748643
301 477482518710752
43 13699538579257
341 697039521327242
398 370576873165770
247 138973556075347
147 341668782491402
177 236908127627083
257 684907838029231
123 806741989308574
330 881922018198672
56 196962249939422
207 537842398089939
182 592705644092238
242 646911424614230
73 617866816803417
230 372798832224659
127 950598348519564
58 47634623217365
347 523222835747001
168 131435094216981
260 26229855268853
147 930414790540614
51 892495413809119
154 8884580656769
376 321081851942261
360 497810150867897
175 208580410615828
153 507509749328025
131 593341318367925
239 151980523805157
86 256718497733467
334 205272135205743
119 996569710302033
244 71867078429896
12 625467395845535
370 831793345188540
263 998433504792366
172 906840067804048
161 213126420181
53 901498741901054
43 74895795869006
173 273321615800993
115 729269042043986
54 113560188952564
60 97581185259209
387 399633003004836
81 830461664103016
95 356672666998522
248 344392489273364
337 184407586968222
304 876937606961074
164 875541915268430
162 512462775763680
306 870776719414739
155 278727844433332
78 479846986545338
174 919300921544887
55 124130343046055
10 263065835127812
315 351616766420091
148 467971354342405
213 644364535894647
48 592364746982178
149 605919075357239
307 210996184084699
68 465387014292374
170 371278386253756
135 178361534950787
255 475823215542632
56 844381849915066
267 851285282741473
291 373461187730852
24 484405061589045
382 594208364065209
130 512332282614828
130 579919245702185
74 312219695071151
216 470244462657144
256 707443038548167
390 568313266731712
44 137550242532934
104 502706184522894
130 552434345305432
238 774769667982963
86 826104489637662
282 28579276783778
263 204777835164222
327 885804844104469
223 33503031755196
1 112749113883240
111 375506958220419
176 243903909706721
203 328045469698976
342 56807268862745
311 201161637598577
124 359175603025851
75 746261819725200
298 814187743606865
228 804410798565197
235 941146626476490
264 566479577121543
248 564765386520977
234 647209500660711
318 229522442680852
371 414740207081646
254 958536527034918
277 235965199853000
286 401262532538484
305 233346164614175
109 982353994287566
241 820115012595429
286 89650259855322
297 439630773658067
151 846082134450526
163 955401601973000
256 611188463306374
21 323415553825893
7 548540612771361
194 867615485798550
149 405111872727759
9 213179244328782
174 928870351686585
47 346139514124628
267 589452011915576
62 120911794379511
190 102284963208561
246 458406181176484
52 808718051075498
228 314849791899176
164 137868952486443
112 207817625910090
142 237336205442171
272 276858842887822
317 321365576115917
366 101443270529130
206 647599687436921
361 755647449630264
7 426940680101834
105 160440940817051
354 578835900178836
355 351678178929476
310 68552334901653
281 826226755810313
173 616222557401364
13 831706284812829
234 844154618812712
4 719307507304406
218 203181361782521
166 496980926672290
285 728314587423245
157 154943765056618
291 845108695277796
378 403311988305537
145 767930329018779
271 942283125455329
245 9211837553738
131 622126506598966
216 525740065524391
367 21508390879310
107 517117544956469
190 803990335912358
317 668864454314357
35 856082785168580
376 851893315079103
276 475498117592070
148 23909818063048
224 649430188402676
243 107390561261238
167 294757465565259
368 517847428982427
283 553900916751142
38 873999259486577
179 57056821670383
284 306630702018053
218 591902196102670
237 944699406660832
291 622608201970733
390 299509756804868
26 778195934119919
220 529389654963636
169 935210430500473
343 24856390192867
340 133810552779757
383 126438384231239
217 101650423044668
307 499467133574320
385 970228444061494
20 509675513765255
173 509904372496603
87 609393200143557
62 747554814159020
111 139050261410262
79 665086476795455
158 591287357699211
361 938233247484083
168 394763018611294
392 83914715008952
35 65999880226380
341 76095316477018
295 488345501875230
78 916726407667517
297 312621006397123
76 393315148583570
295 53748959893677
139 4861047563100
188 553995239515541
68 848672092404759
40 917536228514666
127 255768342553061
29 793773792012513
151 995870886098164
172 230583898195058
316 12412338424682
46 812140008768893
255 675530963294400
116 985021940591201
354 315206778120312
193 736696480599562
88 505275116107448
241 582879290528556
118 252374297352018
82 324312465271078
121 500416196528920
337 152448492185877
88 277655364896223
355 84803505574109
155 8891147112621
45 872336033324534
324 947658373688946
379 193880748995669
257 279745065841413
189 431472969893858
325 841695284279634
309 867019150731163
155 54013894469029
248 162581861222838
394 829529046717230
160 76378703883715
362 354159320840458
11 316131217459832
81 578481199250816
206 122232862187042
139 647721227978490
165 270942736572937
128 619461458137141
36 912214750477995
309 369570032953525
186 594070000790621
141 15055312175956
18 486792518813024
179 293156122124547
202 739485346683334
245 220767760130686
108 808855049712051
35 174081635972249
120 854075301505251
283 161604516376886
212 963320682988896
282 279886609078049
329 415337026848156
257 787585213766330
270 316068814193219
10 729553106422115
287 971930065176023
318 904857152747237
250 266223459406677
1 285063137564706
102 883711780016560
83 353234224171743
387 903333749334010
159 213730910202764
231 498588273068453
398 886613631562931
218 914030372699449
28 30882529457277
173 561440098406535
384 21071710154416
327 464655591860343
399 958204170140122
301 763943062543787
93 415517559272662
395 241458546395612
177 113610146896716
307 659679779791199
153 406127480877221
347 562044924873198
167 938064367462939
259 111762868064087
365 682730662871708
338 22624279993138
153 524663536808485
31 49709870946581
389 790275057246368
196 52334844720553
37 792403868142017
257 725194613519978
165 49218743189740
123 425369848789944
158 479180433198302
281 299170028271824
261 906741010164553
38 108716917392027
128 268711515858655
396 55607343689783
199 738543816849440
77 677337067483584
379 386186950369406
334 713580880446043
297 204828797878826
260 521509830307740
353 9336652124866
185 127733967609839
71 755451895929895
270 297023112982587
258 736503357114841
157 332422232592410
50 474438418561456
6 68790661956513
108 670810018288813
106 966561050904936
216 365445281548406
114 346328309885145
89 856754640696271
148 936168478921478
235 466084909328825
27 973887591756229
109 705992087568807
179 23726169251176
61 312134697514121
211 916898947823512
30 552938871082407
121 915377803645479
192 737913637851261
228 42099978327776
357 452668868197864
295 187150017886051
49 861987371868308
61 376230078011749
273 60694481021084
390 285485662402889
365 885934840666851
355 295226961452750
162 155705031783445
147 102176586282321
388 889454838082316
386 654467235425941
260 420727556191533
191 202777582850657
275 100656252775737
137 154524828910097
74 338861603257750
125 189733169866035
352 177311914137388
66 200897341361298
135 475628141149635
281 202989250162683
30 748014055316139
75 547758144451289
337 496260485165774
267 612798491942755
161 462824620283057
83 909789672301322
124 420201971744932
220 289581965756057
360 585271288256538
304 12632082439039
261 815876494747492
43 119836265430831
55 537873575605467
2 771091875737262
262 345600727235123
20 84139846736105
344 13361554781606
96 963806997763405
222 113651595867507
102 303337943502118
19 164151902097455
194 736331043294834
243 620152615745569
24 832761215706819
276 436477489011892
39 297198784357286
368 548072919764718
394 511140252676665
124 895029105539365
200 353786771773592
237 65540408031097
360 897049677454750
310 475035876658759
369 38327406306520
4 908311273256426
24 520771210966783
210 585632184134638
109 628192304914885
119 601314681244362
277 447969923639041
134 988951219672591
310 608059643544432
352 785452753051079
326 446582584510057
176 874199243894288
126 298839987596348
129 855459603608336
227 338190094658429
195 571828189607096
234 307349133271346
139 438871113307532
130 329722053504449
379 41598994755438
254 863510153544773
25 62958420359228
217 224645124856925
198 444304485676451
134 220829965727539
397 772317808916284
124 516563991222093
60 490227491168275
335 911830312272727
232 306509499555079
203 989681680411164
330 472861700119488
342 825484870568846
313 606762008606162
89 919839159709843
245 414330877215664
70 200201204555855
139 20849494578802
152 581780679803488
31 902676038018046
4 897036583455456
324 318001039223700
230 654165755189315
38 542688705086716
54 571798442568226
229 618896887321232
67 821901688013373
46 636648466396438
349 830500064711281
208 884663552657488
318 157267703024141
97 805123245655910
389 464977968831495
20 640039321376697
39 969362708994023
204 491401642935376
153 448835579176339
146 797789782717545
89 418192950763618
315 633394996260127
387 452850899620344
222 533120413152680
120 248131374672781
126 150990921657755
282 835476529985176
14 299380353954907
251 304675631114788
383 351614835950686
202 479733998771070
333 145095222325930
288 140180797455425
318 584746275657554
128 25532278949564
302 763786371723987
382 603942029405144
154 809470049668250
215 487259577661869
313 872399019812058
120 514365529101436
292 304556109560228
265 779693971152298
313 211435833947965
303 471770123137611
259 206070557076012
368 588273820611460
98 452246064774867
281 20097275228872
297 815445866703704
54 30525922102171
330 489275387391379
90 131719047439970
240 349310728561354
369 556608995311720
200 60984701669105
332 527253814677865
39 956275055651839
328 519554744753580
18 484065675758122
386 582328747721006
152 901713325270191
225 861291614288040
108 163534471823780
253 872401401933087
88 845604634096707
360 51216980651636
44 367836911730494
166 739868268102198
61 410620438738242
12 31915932451454
315 587335951877605
98 693507508061421
388 317714345986809
365 714844278368122
116 166718811287285
342 163047294879280
59 261011999430163
213 840563346778562
333 397978809182526
240 970576123800163
381 919879660116168
81 668197623531937
396 862938026936918
263 838179486651844
126 548426715192994
28 357005077454111
197 931442904266872
359 117794433675217
268 745221785120562
55 390083865603523
396 218689929774710
330 954397311205148
317 275964692586814
353 312835325136468
384 5330448175783
308 777448910486223
47 263943248274040
73 403441996333903
364 202626495622588
174 145254597562083
115 51869190364401
53 283884548827477
3 545672139633587
84 644690315397859
272 557881105739883
114 4105299126366
256 198898676697329
97 210520882974696
118 798610870471282
247 17889194944728
171 619083585675557
177 897493805599899
101 57489671665224
363 41772050637285
97 559255388748369
65 406396963086413
205 492573563410215
217 782692553966725
148 278659324021277
115 857356225328218
149 993758916471417
371 884919115827625
293 29160321087408
10 492473462136595
142 483126734118958
372 668644053798436
80 490717122411242
134 518616947909427
320 587271590597049
59 385071623236525
199 216056120822853
18 606506794483635
53 764107600641225
33 377534567057237
305 833339742745164
114 910122960112120
178 689590410862841
349 244826113161206
261 944277614942955
39 708484609103284
61 775341649008383
287 404415781105839
2 880922085250561
118 907279874837077
325 475002314349950
21 203458183186078
101 485949441600497
69 639358616497141
36 288349071082179
172 673234475889839
248 882449509163920
127 327897977728786
383 221822505289409
203 392829141766482
231 139000482220689
112 958989646422921
345 208948932998410
83 141547080739739
33 355330068989764
316 264418090901965
254 934909004112563
294 245772252035827
119 574396262449382
40 36707581923751
351 390792333949646
350 685392102268927
45 56115735452290
358 892094594980922
96 355828016429609
158 22133600971559
143 78719279904800
260 578814243028911
387 132408502353364
158 902742972907574
19 562434428787297
350 978026773854889
150 957544907320918
358 367063439295991
382 413735824565896
240 712481207357867
360 207749638425258
300 704604174413617
92 790335594855827
112 933954942725080
399 874587267040244
245 106766436224557
342 807565647000032
198 627583632511270
395 213274100636188
143 269923417055550
102 632099949049679
1 686941321420545
127 35963828112984
337 40238902328631
362 812712590050408
256 714707074898287
19 365929568268634
