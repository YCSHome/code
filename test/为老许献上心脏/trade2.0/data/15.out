7552771921774721
